### PR TITLE
Tm 832 activity restriction

### PR DESCRIPTION
**Pull request by robline**

### CODE REVIEW

File: timepiece/entries/forms.py [hunk 1]
```diff
@@ -4,15 +4,21 @@
...
 from timepiece import utils
 from timepiece.crm.models import Project, ProjectRelationship
 from timepiece.entries.models import Entry, Location, ProjectHours
+from timepiece.entries.lookups import ActivityLookup
 from timepiece.forms import (
     INPUT_FORMATS, TimepieceSplitDateTimeField, TimepieceDateInput)
 
 
 class ClockInForm(forms.ModelForm):
+    activity = selectable.AutoCompleteSelectField(
```
Comment (daaray) [inline timepiece/entries/forms.py]: This should be an `AutoComboboxSelectWidget`.  You can configure the widget in the class `Meta`:

```
        widgets = {
            'activity': selectable.AutoComboboxSelectWidget(lookup_class=ActivityLookup),
        }
```

File: timepiece/entries/lookups.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from selectable.base import ModelLookup
+from selectable.registry import registry
+
+from timepiece.crm.models import Project
+from timepiece.entries.models import Activity
+
+
+class ActivityLookup(ModelLookup):
```
Comment (daaray) [inline timepiece/entries/lookups.py]: Can you add a test to ensure the lookup is returning the expected option(s)?

File: timepiece/entries/forms.py (hunk 1 above)
Comment (daaray) [inline timepiece/entries/forms.py]: FYI: Moving to this widget will require some additional CSS tweaks for the dropdown button

https://django-selectable.readthedocs.org/en/version-0.7.0/advanced.html#using-with-twitter-bootstrap

**Comment by robline** [comment]
good points!  addressed PR feedback

File: timepiece/entries/forms.py [hunk 3]
```diff
@@ -4,15 +4,20 @@
...
 from timepiece import utils
 from timepiece.crm.models import Project, ProjectRelationship
 from timepiece.entries.models import Entry, Location, ProjectHours
+from timepiece.entries.lookups import ActivityLookup
 from timepiece.forms import (
     INPUT_FORMATS, TimepieceSplitDateTimeField, TimepieceDateInput)
 
 
 class ClockInForm(forms.ModelForm):
+    activity = selectable.AutoComboboxSelectWidget(
```
Comment (daaray) [inline timepiece/entries/forms.py]: With the widget definition in the `Meta` blow, you can remove this field definition entirely.

**Pull request merged**